From: Jan Engelhardt <jengelh@inai.de>
Date: 2014-02-05 03:01:02.110857012 +0100

Kill __DATE__ from source, it's pointless and can cause rebuilds.

---
 src/language/paricfg.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: pari-2.11.0/src/language/paricfg.c
===================================================================
--- pari-2.11.0.orig/src/language/paricfg.c
+++ pari-2.11.0/src/language/paricfg.c
@@ -17,6 +17,6 @@ const char *paricfg_version = PARIVERSIO
 const char *paricfg_buildinfo = PARIINFO;
 const long  paricfg_version_code = PARI_VERSION_CODE;
 const char *paricfg_vcsversion = PARI_VCSVERSION;
-const char *paricfg_compiledate = __DATE__;
+const char *paricfg_compiledate = "openSUSE";
 const char *paricfg_mt_engine = PARI_MT_ENGINE;
 const char *paricfg_gphelp = GPHELP;
